src/composables/transactionList.ts

```diff
@@ -30,15 +32,18 @@ interface UseTransactionListOptions {
  * if the cached list should be cleared and fetched again.
  */
 const state = useStorageRef<{
-  accountAddress?: AccountAddress;
-  assetContractId?: AssetContractId;
+  accountAddress: AccountAddress;
+  assetContractId: AssetContractId;
   isEndReached: boolean;
   isInitialLoadDone: boolean;
   networkName?: string;
   nextPagePaginationParams: ITransactionApiPaginationParams;
   transactionsLoaded: ICommonTransaction[];
+  protocol?: Protocol;
 }>(
   {
+    accountAddress: '',
+    assetContractId: '',
```

Comment:
I don't see any reason to create falsy values here. What kind of benefit it is giving then using `undefined`?